src/backend/web/handlers/team.py

```diff
@@ -117,5 +120,31 @@ def team_list(page: int) -> str:
         "page_labels": page_labels,
         "cur_page_label": cur_page_label,
         "current_page": page,
+        "current_season": SeasonHelper.get_current_season()
     }
     return render_template("team_list.html", template_values)
+
+
+@cached_public
+def avatar_list(year: Optional[Year] = None) -> Response:
+    if not year:
+        return redirect(url_for('avatar_list', year=SeasonHelper.get_current_season()))
```

Comment:
Do we need to redirect? Or could we just set year to be the current year?

We could also do this using default arguments in the function declaration (that way it's never None)

Reply:
- Yeah, I agree it makes more sense to just render the current year.

- I actually tried that, but it started yelling `AssertionError: No api proxy found for service "memcache"` at me (!!!)

Reply:
Lemme checkout and see what's up with the second bullet point. I'm unsure why that would be an issue. Where were you seeing the assertion failure? In tests?

Reply:
Oh right yes - I understand this. `SeasonHelper.get_current_season` hits a `Sitevar` (backed by ndb). ndb for the instance isn't setup when the constructor is called.

All this is to say - I think the `year = year or ...` code  you have is great and the right way to go here :+1: